tests/fortex/health/processors/temporal_mention_normalizing_processor_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for ScispaCyProcessor
+"""
+
+import unittest
+
+from forte.data.data_pack import DataPack
+from forte.data.readers import StringReader
+from forte.pipeline import Pipeline
+
+from ftx.medical.clinical_ontology import NormalizedTemporalForm
+from fortex.health.processors.temporal_mention_normalizing_processor import (
+    TemporalMentionNormalizingProcessor,
+)
+
+
+class TestTemporalMentionNormalizingProcessor(unittest.TestCase):
+    def setUp(self):
+        self.nlp = Pipeline[DataPack](enforce_consistency=True)
+        self.nlp.set_reader(StringReader())
+        config = {
+            "entry_type": "ft.onto.base_ontology.Document",
+            "model_name": "en_core_web_sm",
+            "pipe_name": "timexy",
+        }
+        self.nlp.add(TemporalMentionNormalizingProcessor(), config=config)
+        self.nlp.initialize()
+
+    def test_TemporalMentionNormalizingProcessor(self):
+        sentences = ["10.10.2010", "10 days ago", "5 years later"]
```

Comment:
Let's have actual sentences here, instead of exact matches of what we expect to identify from the processor.

Reply:
actually Timexy model that we are using here is not good for detection. Idea was that BERT based model will detect the temporal phrases in tagging_processor then this normalizing processor will be the next pipeline stage.

Reply:
Lets add tests that depict that kind of a pipeline structure too then? So, we know for sure our use case is being handled as its supposed to.